ssg/build_yaml.py

```diff
@@ -166,37 +166,28 @@ def dump_yaml(self, file_name, documentation_complete=True):
 
     def _parse_selections(self, entries):
         for item in entries:
-            if "." in item:
-                rule, refinement = item.split(".", 1)
-                property_, value = refinement.split("=", 1)
-                if property_ not in XCCDF_REFINABLE_PROPERTIES:
-                    msg = ("Property '{property_}' cannot be refined. "
-                           "Rule properties that can be refined are {refinables}. "
-                           "Fix refinement '{rule_id}.{property_}={value}' in profile '{profile}'."
-                           .format(property_=property_, refinables=XCCDF_REFINABLE_PROPERTIES,
-                                   rule_id=rule, value=value, profile=self.id_)
-                           )
-                    raise ValueError(msg)
-                self.refine_rules[rule].append((property_, value))
-            elif "=" in item:
-                varname, value = item.split("=", 1)
-                self.variables[varname] = value
-            elif item.startswith("!"):
-                self.unselected.append(item[1:])
-            else:
-                self.selected.append(item)
```

Comment:
Extracted to `apply_selection`, again to allow extension of parsing `selections:` in derived classes.